pages/guides/receive-money/add-funding-source.mdx

```diff
@@ -8,141 +8,29 @@ meta:
-        }, function(err, res) {
-    console.log('Error: ' + JSON.stringify(err) + ' -- Response: ' + JSON.stringify(res));
-  });
-});
-</script>
+## Bank Addition and Verification Methods
 
-```
+There are currently two ways of adding a bank to a Customer with the Dwolla API. A simplified table below outlines the similarities and differences of each method.
 
-The Customer will complete the IAV flow by authenticating with their online banking credentials. You’ll know their bank account was successfully added and verified if you receive a JSON response in your callback that includes a link to the newly created funding source.
+| Bank Addition Method                                                                                                  | Will the bank be verified?    | Required Information            |
+| --------------------------------------------------------------------------------------------------------------------- | ----------------------------- | ------------------------------- |
+| API - Account & Routing Number                                                                                        | Optional - With Microdeposits | Bank Account and Routing Number |
+| Third Party - Plaid ([Example](https://github.com/Dwolla/integration-examples/tree/main/dwolla-plaid-funding-source)) | Yes                           | Online Bank Credentials         |
 
-<Image
-  src={iavSelectAccount}
-  alt="IAV select account screen"
-  width="75%"
-  height="75%"
-/>
+## Step 2A: Add a Bank to an Unverified Customer
 
-You’ll know their bank account was successfully added and verified if you receive a JSON response in your callback that includes a link to the newly created funding source.
+In this step, we will create and attach a verified funding source to your Customer, which will be done using Plaid Link, a third-part bank verification provider. This method will give your Customers the ability to add and verify their bank account in a matter of seconds by authenticating using their online banking credentials.
 
-### Example success callback
+Once your Customer reaches the page in your application to add a bank account, you will first use Plaid Link to authenticate the user's bank account, exchange the `public_token` for a `processor_token`, and then send the `processor_token` to Dwolla to create the funding source.
 
-```javascript
-{"_links":{"funding-source":{"href":"https://api-sandbox.dwolla.com/funding-sources/80275e83-1f9d-4bf7-8816-2ddcd5ffc197"}}}
-```
+To integrate Dwolla with Plaid, we recommend checking out our [Add a Bank via Dwolla + Plaid Integration](/guides/plaid) guide. Additionally, if you would like to see a working example that creates a Dwolla funding source and instantly verifies it via Plaid Link (using Next.js/React), we recommend checking out our [dwolla-plaid-funding-source](https://github.com/Dwolla/integration-examples/blob/main/dwolla-plaid-funding-source) integration example on our GitHub profile.
```

Comment:
Apologies for overlooking this earlier. Can we change the second sentence to the following? 
> Additionally, if you would like to see a working example that verifies a bank via Plaid Link (using Next.js/React) and attaches it as a verified funding-source to a Dwolla Customer, please check out our [dwolla-plaid-funding-source](https://github.com/Dwolla/integration-examples/blob/main/dwolla-plaid-funding-source) integration example on our GitHub profile.